codebuild/spec/buildspec_sanitizer.yml

```diff
@@ -75,6 +75,13 @@ batch:
         variables:
           S2N_LIBCRYPTO: openssl-3.0
           COMPILER: gcc
+    - identifier: gcc_openssl_3_fips
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: 024603541914.dkr.ecr.us-west-2.amazonaws.com/docker:ubuntu24_20250214
```

Comment:
We can drop the date in a future PR, don't forget!

Reply:
We'll drop the whole image tag in a later PR ;) I only need it because the default image used by this build doesn't have working openssl-3.0-fips yet.